src/pd/mod.rs

```diff
@@ -102,3 +102,30 @@ pub trait PdClient: Send + Sync {
     // Report pd the split region.
     fn report_split(&self, left: metapb::Region, right: metapb::Region) -> Result<()>;
 }
+
+use futures::BoxFuture;
+
+pub type PdFuture<T> = BoxFuture<T, Error>;
+
+pub trait AsyncPdClient: Send + Sync {
```

Comment:
I see a PDAsyncClient and AsyncPdClient, can we use a better name to distinguish them?